lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -553,6 +553,31 @@ public function testImageTagWithTheme() {
 		}
 	}
 
+	/**
+	 * testBase64ImageTag method
+	 *
+	 * @return void
+	 */
+	public function testBase64ImageTag()
+	{
+		$this->Html->request->webroot = '';
+		
+		$result = $this->Html->image('cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+		$result = $this->Html->image('/img/cake.icon.png', ['base64' => true]);
+		$this->assertTags($result, array(
+			'img' => array(
+				'src' => 'preg:/data:image\/png;base64,(?:[A-Za-z0-9+\/]{4})*(?:[A-Za-z0-9+\/]{2}==|[A-Za-z0-9+\/]{3}=|[A-Za-z0-9+\/]{4})/',
+				'alt' => ''
+			)));
+
+	}
```

Comment:
- If you access an image that is not there cakephp will throw a 404 exception and by default log it (this, however, can be disabled)
- As it is a new option I don't see a problem throwing exceptions - it is not "all of a sudden". It also makes sure images render where you would normally never knew if they did, because it wouldn't even appear in the logs.